src/hooks/private/useCheckbox/README.md

```diff
@@ -0,0 +1,26 @@
+| name           | Name                        |                                                                          `string`                                                                          |         |
+| value          | Value                       |                                                                          `string`                                                                          |         |
+| checked        | Checked flag                |                                                                         `boolean`                                                                          |         |
+| defaultChecked | Default checked value       |                                                                         `boolean`                                                                          |         |
+| indeterminate  | Indeterminate flag          |                                                                         `boolean`                                                                          |         |
+| controlRef     | Ref-link on control element |                                                                        `React.Ref`                                                                         |         |
+| controlProps   | Another control props       | `Omit<React.InputHTMLAttributes, 'name', 'value', 'id', 'onFocus', 'onBlur', 'disabled', 'type', 'onChange', 'defaultChecked', 'checked', 'aria-checked'>` |         |
+| disabled       | Disabled flag               |                                                                         `boolean`                                                                          |         |
+| onUpdate       | OnUpdate callback           |                                                                `(checked: boolean) => void`                                                                |         |
+| onChange       | OnChange callback           |                                                            `(event: React.ChangeEvent) => void`                                                            |         |
+| onFocus        | OnFocus callback            |                                                            `(event: React.FocusEvent) => void`                                                             |         |
+| onBlur         | OnBlur callback             |                                                            `(event: React.FocusEvent) => void`                                                             |         |
+| id             | ID attribute                |                                                            `(event: React.FocusEvent) => void`                                                             |         |
+
+## Result
+
+- checked, radio-input status `boolean`
```

Comment:
Let's use md table here, checked - Checked state, inputProps - Props to pass to the input element